components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -94,18 +94,30 @@
                 <div class="healthcheck-header {{service.health_check}}">
                   Health Check Error Message
                 </div>
-                <div class="healthcheck-content">
+                <div class="healthcheck-content" [class.active]="activeErrorAccordions.includes(i)">
                   {{service.health_check_result.stderr}}
+                  <div class="show-more-container">
+                    <button 
+                    *ngIf="service.health_check_result.stderr.length > 138"
```

Comment:
UX requested ~140 characters in the short form of the output.  By my calculation - 138 characters is set to 3 lines of output.

Reply:
good point - thanks for the suggestion!